client/state/data-layer/wpcom/theme-filters/index.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * External dependencies
+ */
+import i18n from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import {
+	THEME_FILTERS_REQUEST,
+	THEME_FILTERS_ADD,
+} from 'state/action-types';
+import { dispatchRequest } from 'state/data-layer/wpcom-http/utils';
+import { http } from 'state/data-layer/wpcom-http/actions';
+import { errorNotice } from 'state/notices/actions';
+
+const fetchFilters = ( { dispatch }, action ) => {
+	dispatch( http( {
+		method: 'GET',
+		apiVersion: '1.2',
+		path: '/theme-filters',
+	}, action ) );
+};
```

Comment:
fun fact! the curlies are redundant here. arrow functions which don't have them evaluate the expression and then return the value. if our function is nothing more than another function call we can skip them (opinions differ but I personally prefer eliminating them - helps me quickly read the code)

```js
const fetchFilters = ( … ) => dispatch( http( {
	method: 'GET',
	apiVersion: '1.2',
	path: '/theme-filters',
}, action ) );
```